### PR TITLE
Adds systemctl.enable stanza to systemd units

### PR DESCRIPTION
**Pull request by chuckbutler**
The services were started but will not survive unit interruption, reboots,
etc. as the systemd services were not enabled on boot

### CODE REVIEW

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 1]
```diff
@@ -521,11 +521,10 @@ def render_files():
     render('kube-defaults.defaults', '/etc/default/kube-defaults', context)
 
     # when files change on disk, we need to inform systemd of the changes
-    try:
-        check_call(['systemctl', 'daemon-reload'])
-    except CalledProcessError:
-        # we failed, chances are no changes were made. so assume this is fine
-        pass
+    call(['systemctl', 'daemon-reload'])
+    call(['systemctl', 'enable', 'kube-apiserver'])
```
Comment (mbruzek) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: You are not calling the new method that you created. Clearly an oversight. But change these three lines to:
```python
_enable_services(['kube-apiserver', 'kube-controller-manager', 'kube-scheduler'])
```

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 2]
```diff
@@ -521,11 +521,10 @@ def render_files():
     render('kube-defaults.defaults', '/etc/default/kube-defaults', context)
 
     # when files change on disk, we need to inform systemd of the changes
-    try:
```
Comment (mbruzek) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: Did we eliminate the need for a try/except?  with this new code?

Comment (chuckbutler) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: Yeah, its performing the same function as this block with check_call

by piping it through call we can just run it and if it fails subprocess didnt five a fig. so why do more loc?

**Comment by chuckbutler** [comment]
Hey matt, i updated the PR. can you PTAL?

**Comment by mbruzek** [comment]
The redactor looks good. I like the # comment you put in kubernetes-worker, but is missing from kubernetes-master. Not a blocking issue though.

+1 Looks good, great find and addition to this code.

**Pull request merged**